django/db/backends/postgresql/operations.py

```diff
@@ -253,7 +253,9 @@ def subtract_temporals(self, internal_type, lhs, rhs):
         if internal_type == 'DateField':
             lhs_sql, lhs_params = lhs
             rhs_sql, rhs_params = rhs
-            return "age(%s, %s)" % (lhs_sql, rhs_sql), lhs_params + rhs_params
+            # This can be replaced with make_interval() when only
```

Comment:
Might want to mention the whole `make_interval(...)` SQL it should be replaced with.